spec.html

```diff
@@ -27188,14 +27188,14 @@ <h1>The AggregateError Constructor</h1>
           <h1>AggregateError ( _errors_, _message_ )</h1>
           <p>When the *AggregateError* function is called with arguments _errors_ and _message_, the following steps are taken:</p>
           <emu-alg>
-            1. If NewTarget is *undefined*, let _newTarget_ be the active function object, else let _newTarget_ be NewTarget.
-            1. Let _O_ be ? OrdinaryCreateFromConstructor(_newTarget_, `"%AggregateError.prototype%"`, &laquo; [[ErrorData]] &raquo;).
+            1. If NewTarget is *undefined*, let _newTarget_ be the active function object; else let _newTarget_ be NewTarget.
```

Comment:
is this a convention we follow? my gut reaction is that we always use commas.

Reply:
This is a convention we follow. This is the only instance of ", else" in the spec, compared to 74 instances of "; else".